src/Utilities/Compiler/DiagnosticCategoryAndIdRanges.txt

```diff
@@ -19,6 +19,7 @@ Naming: CA1700-CA1727
 Interoperability: CA1400-CA1419
 Maintainability: CA1500-CA1509
 Reliability: CA9998-CA9999, CA2000-CA2018
+Correctness: CA3200, RS1000-RS1999
```

Comment:
I don't think we need to introduce a new category. It's likely that the category is unintentionally wrong written in the runtime issue. cc @bartonjs

Reply:
Agreed, please don't add this category. Also RS* rules do not belong here.

Reply:
@Youssef1313 @mavasani Okay. Which category should I use? Reliability?

Reply:
Reliability sounds fine to me.